src/popup/modules/QrLib/QrError.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Errors in QR code generation
+ *
+ * @module QrLib/QrErrors
+ */
+
+export class DataOverflowError extends Error {
+    constructor(...params) {
+        super(...params);
+        this.message = "data overflow error";
```

Comment:
So when we are at it: Can you change it to only _optionally_ use this as a default message? If one manually does `new DataOverflowError("custom message")` that should still be honored and not being dismissed.